connectors/jetty-connector/src/main/java/org/glassfish/jersey/jetty/connector/JettyConnector.java

```diff
@@ -45,6 +45,7 @@
 
 import javax.net.ssl.SSLContext;
 
+import org.eclipse.jetty.client.AuthenticationProtocolHandler;
```

Comment:
This one does not seem to be needed here

Reply:
Fixed